compiler/include/TryStmt.h

```diff
@@ -26,25 +26,31 @@ class TryStmt : public Stmt
 {
 
 public:
+
+  static BlockStmt*   build(bool tryBang, Expr*      expr);
   static BlockStmt*   build(bool tryBang, BlockStmt* body);
+  static BlockStmt*   build(bool tryBang, BlockStmt* body, BlockStmt* catches);
 
-                      TryStmt(bool tryBang, BlockStmt* body);
+                      TryStmt(bool tryBang, BlockStmt* body, BlockStmt* catches);
                       ~TryStmt();
   BlockStmt*          body() const;
   bool                tryBang() const;
 
   void                accept(AstVisitor* visitor);
-  Expr*               copy(SymbolMap* map = NULL, bool internal = false);
-  Expr*               copyInner(SymbolMap* map);
   void                replaceChild(Expr* old_ast, Expr* new_ast);
   Expr*               getFirstChild();
   Expr*               getFirstExpr();
   Expr*               getNextExpr(Expr* expr);
   GenRet              codegen();
+  DECLARE_COPY(TryStmt);
+
+  BlockStmt*          _body;
```

Comment:
Yeah, this is disappointing, but piping AST_CHILDREN_CALL through accessor methods was brittle and we'd like to stay consistent. We can fix it in v2. :)